int/tests/conftest.py

```diff
@@ -168,3 +169,55 @@ async def http_endpoint(agent: BaseAgent):
     with suppress(asyncio.CancelledError):
         await server_task
     await agent.cleanup()
+
+
+@pytest.fixture
+async def make_did():
+    """DID factory fixture"""
+
+    def _make_did():
+        did = create_did.asyncio(client=backchannel)
+        return did
+
+    yield _make_did()
```

Comment:
We need to return a reference to the `_make_did` method rather than the result of calling the `_make_did` method:
```suggestion
    yield _make_did
```